go.mod

```diff
@@ -78,18 +79,18 @@ require (
 	github.com/prometheus/common v0.9.1
 	github.com/prometheus/tsdb v0.8.0 // indirect
 	github.com/rancher/dynamiclistener v0.2.1-0.20200418023342-52ede5ec9234
-	github.com/rancher/kontainer-engine v0.0.4-dev.0.20200512184531-c395feca5b3b
+	github.com/rancher/kontainer-engine v0.0.4-dev.0.20200327180925-6d88ca2af46f
```

Comment:
Need to update this after merging https://github.com/rancher/kontainer-engine/pull/229 with the commit from the release/v2.4 branch